src/goRouter.php

```diff
@@ -75,8 +76,11 @@ public function route() {
             $this->route = self::ROUTE_NAME_HOME;
         } elseif ($this->pathInfo === self::ROUTE_PATH_API) {
             $this->route = self::ROUTE_NAME_API;
-        } elseif (preg_match('#^([^/]+)\.qr$#', $this->pathInfo, $matches)) {
```

Comment:
rather than removing the .qr route, should it be kept and return the .png? Probably not needed unless someone is hotlinking to the .qr file.